app/controllers/wishlist_items_controller.rb

```diff
@@ -7,7 +7,7 @@ def create
     if @wishlist_items.new_record?
       @wishlist_items.save
       flash[:notice] = 'Product wishlisted'
-      redirect_to wishlist_path
+      redirect_to wishlist_path(current_user)
```

Comment:
you are alredy use this path redirect_to wishlist_path

Reply:
check carefully, I'm not repeating the code it's been overwritten